curiouslearning/FeedTheMonsterJS
# fix: CR-180 amharic rebuild

# Changes
- lang/amharic/ftm_amharic.json

# How to test
- When I play the development APK and open Amharic,

Then I should see the title as “ጓደኛችሁን መግቡ”.

Ref: [CR-180](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-180)


[CR-180]: https://curiouslearning.atlassian.net/browse/CR-180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
